subsys/lorawan/radio/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+zephyr_sources_ifdef(CONFIG_SX1272 sx1272.c)
```

Comment:
I don't think you need a CMakeLists.txt here, as all the source code is in ext/...

Reply:
Are you suggested, I should move all imp func to ext/ dir ?

Reply:
@miyatsu Sorry I got confused with the patches. Actually the driver function implementation (sx1272.c and sx1276.c), should go in drivers/lora with this CMakeLists.txt and the radio Kconfig part. Looks like it only needs to be moved, content seems fine to me. Maybe change the Kconfig first config option to:
menuconfig LORA
    bool "LoRa Radio chip support"
    default n
    select <the lorawan L2 option>

if LORA
... the config option for the 2 radio chip (the separation in dedicated Kconfig.sx--- as you did is nice. Maybe add the prefix LORA_ to all these options though. This is usually how we do, so it tells to which part the config option belongs to.

endfi #LORA